### PR TITLE
Fix demo site not being able to load.

### PR DESCRIPTION
**Pull request by cliffkoh**
My PR in #6766 was a little sloppy and made things worse. Sorry about that. This should fix things. Tested across multiple packages. Will verify with the PR deploy too.

### CODE REVIEW

**Comment by cliffkoh** [comment]
OMG copy and paste mistake.. submitting a fix for stupid mistake again

File: apps/fabric-website-resources/index.html [hunk 1]
```diff
@@ -22,12 +22,12 @@
       // }
     };
 
-    var isProduction = (
-      (document.location.search.indexOf('localhost') === -1 && document.location.search.indexOf('127.0.0.1') === -1)
-      || (document.location.search.indexOf('?min=0') === -1 && (
-        document.location.search.indexOf('?min=1') > -1 ||
-        location.pathname === '/fabric-react/master/'
-      )));
+    var isLocalHost = (document.location.search.indexOf('localhost') !== -1 || document.location.search.indexOf('127.0.0.1') !== -1);
```
Comment (kenotron) [inline apps/fabric-website-resources/index.html]: .search -> .href

**Comment by cliffkoh** [comment]
Tested with both npm start default and localhost and verified it loads the right set of files. Will test with PR deploy when it is ready.

image

**Comment by cliffkoh** [comment]
PR deploy is broken for an unrelated issue that @kenotron is working on to fix.

**Comment by kenotron** [comment]
Gonna close & reopen here.

**Comment by cliffkoh** [comment]
@kenotron PR deploy is still broken because both demo-app.js and demo-app.min.js is a 404??!

**Comment by cliffkoh** [comment]
@kenotron just fixed a bug in the pipeline so close and reopen to trigger a rebuild.

**Comment by cliffkoh** [comment]
image

**Pull request merged**